website/content/en/preview/contributing/documentation-updates.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Rather than creating a contributing directory, does it make more sense to create a CONTRIBUTING.md file like in https://github.com/kubernetes-sigs/karpenter/blob/main/CONTRIBUTING.md

Reply:
I'm worried that having contribution documentation spread across multiple locations will reduce the chance of users finding the correct documentation.